### PR TITLE
Add Klaviyo SDK recipe

### PR DESCRIPTION
**Pull request by Th3Mouk**
| Q             | A
| ------------- | ---
| License       | MIT

A pretty simple recipe that will simplify the package installation

### CODE REVIEW

File: klaviyo/php-sdk/2.0/config/packages/klaviyo.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+services:
+    Klaviyo\Klaviyo:
+        class: Klaviyo\Klaviyo
```
Comment (Nyholm) [inline klaviyo/php-sdk/2.0/config/packages/klaviyo.yaml:3]: This row is not needed when you have the service ID as: "Klaviyo\Klaviyo"

**Review by Nyholm — CHANGES_REQUESTED** [review]
Thank you. Just a small thing

File: klaviyo/php-sdk/2.0/config/packages/klaviyo.yaml (hunk 1 above)
Comment (Th3Mouk) [reply klaviyo/php-sdk/2.0/config/packages/klaviyo.yaml:3]: I didn't know, thanks for the tip.

**Review by Nyholm — APPROVED** [review]
Thank you

**Pull request merged**